src/java/htsjdk/samtools/BAMFileReader.java

```diff
@@ -67,6 +67,9 @@
     // If true, all SAMRecords are fully decoded as they are read.
     private boolean eagerDecode;
 
+    // If true, the BAMFileReader will use asynchronous IO.
+    private final boolean useAsynchronousIO;
```

Comment:
Add a note here that this field currently has no effect (is not hooked up anywhere), but will be in the future.